esp/templates/program/modules/surveymanagement/manage.html

```diff
@@ -0,0 +1,180 @@
+{% extends "main.html" %}
+
+{% block title %}{{program.niceName}} Survey Administration{% endblock %}
+
+{% block stylesheets %}
+    {{ block.super }}
+    <link rel="stylesheet" href="/media/styles/forms.css" type="text/css" />
+    <link rel="stylesheet" href="/media/styles/expand_display.css" type="text/css" />
+    <link rel="stylesheet" href="/media/styles/survey_manage.css" type="text/css" />
+{% endblock %}
+
+{% block xtrajs %}
+    {{ block.super }}
+    <script type="text/javascript">
+        var question_types = {{ question_types|safe }};
+    </script>
+{% endblock %}
+
+{% block content %}
+<h1>Survey Management for {{ program.niceName }}</h1>
+
+<p>
+You can use this page to set up surveys for teachers or students of the program. You will first need to create a survey, then you can add questions to it.
```

Comment:
Every time I add a question by hitting "create/save," it's overriding the current question on my survey even though I'm trying to add a new question?

Reply:
OK, I think I know what's going on. I'll take a look when I get a chance!

Reply:
Should be all set with the latest commit!